packages/headless/src/controllers/standalone-search-box/headless-standalone-search-box.ts

```diff
@@ -34,7 +33,7 @@ export interface StandaloneSearchBoxProps {
  * The `StandaloneSearchBox` headless controller offers a high-level interface for designing a common search box UI controller.
  * Meant to be used for a search box that will redirect instead of executing a query.
  */
-export interface StandaloneSearchBox extends SearchBox, Controller {
+export interface StandaloneSearchBox extends SearchBox {
```

Comment:
I removed the `Controller` interface because it was not needed, but the parser will now generate the correct output whether or not it is included.